instrumentation/events/lttng-module/lttng-kallsyms.h

```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#undef TRACE_SYSTEM
+#define TRACE_SYSTEM lttng_kallsyms
+
+#if !defined(LTTNG_TRACE_LTTNG_KALLSYMS_H) || defined(TRACE_HEADER_MULTI_READ)
+#define LTTNG_TRACE_LTTNG_KALLSYMS_H
+
+#include <probes/lttng-tracepoint-event.h>
+
+LTTNG_TRACEPOINT_EVENT(lttng_kallsyms_kernel_symbol,
```

Comment:
We might want to split this into two events:
lttng_kallsyms_symbol_core
and
lttng_kallsyms_symbol_module

The "core" event would not have a "module" string at all.